pipeline/luigi/CompileVCFFiles.py

```diff
@@ -1383,6 +1384,17 @@ def run(self):
 
 @requires(FindMissingReports)
 class RunDiffAndAppendChangeTypesToOutput(luigi.Task):
+    def _extract_release_date(self, archive_path):
```

Comment:
What do you think about pulling the release date from the output/release/metadata/version.json file rather than from the tar.gz filename? The version.json file is where data comes from that we display at http://brcaexchange.org/releases for example. In theory they should be the same, but it's possible that a user error could lead to a disparity.

Reply:
Yes, good point, I didn't think of that. I'll fix it.